packages/react-notifications/src/NotificationsModal.tsx

```diff
@@ -7,7 +6,13 @@ export const NotificationsModal = () => {
   const { notifications } = useNotifications();
 
   return (
-    <Modal isOpen={!!notifications} style={{ content: { minWidth: '30%' } }}>
+    <Modal
+      isOpen={!!notifications}
+      style={{ content: { minWidth: '30%' } }}
+      appElement={document.getElementById('root') as HTMLElement}
```

Comment:
What if we don't have the name "root" on our App? Set it only on our tests by now.

Reply:
Done!